rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import resolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+
+export default {
+	input: 'src/index.js',
+	output: {
+		file: 'dist/bundle.js',
+    format: 'cjs',
+    name: 'ObservePropsDecorator',
+		sourcemap: true
+	},
+	plugins: [
+		resolve({
+      customResolveOptions: {
+        moduleDirectory: 'node_modules'
+      }
+    }), // tells Rollup how to find date-fns in node_modules
+    commonjs({ // converts date-fns to ES modules
```

Comment:
why do we need commonjs?

Reply:
We need CommonJS plugin to convert CommonJS modules to ES6, so they can be included in a Rollup bundle.
Without the plugin:
1) rxjs imports and lodash with exact paths aren't handled by Rollup;
2) the final bundle's size is by 3 times bigger than with the plugin.

Thus, the plugin fixes imports and helps with bundle size optimization. I suppose that rxjs, lodash and more other libs are available for use like cjs modules.